package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "20-intro_to_angular_and_webpack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "lint": "eslint *.js app/*.js",
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/maschigokae/20-intro_to_angular_and_webpack.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/maschigokae/20-intro_to_angular_and_webpack/issues"
+  },
+  "homepage": "https://github.com/maschigokae/20-intro_to_angular_and_webpack#readme",
+  "dependencies": {
+    "angular": "^1.6.1",
+    "babel-core": "^6.21.0",
+    "babel-loader": "^6.2.10",
+    "babel-preset-es2015": "^6.18.0",
+    "cowsay-browser": "^1.1.8",
+    "css-loader": "^0.26.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "html-webpack-plugin": "^2.26.0",
+    "node-sass": "^4.2.0",
+    "sass-loader": "^4.1.1",
+    "style-loader": "^0.13.1",
+    "webpack": "^1.14.0"
```

Comment:
You'll need to install Phantom JS to replicate a headless browser, you'll have to be sure to also include it in your karma config after you've installed it .